.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,13 @@
+name: PR
+
+on: pull_request
```

Comment:
I think you want to set the branches the PR can be against, so it is only run against `main`?